keyboards/launchpad/keymaps/default_rgb/keymap.c

```diff
@@ -0,0 +1,61 @@
+// Below layout is based upon /u/That-Canadian's planck layout
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _RGB 1
+#define _FUNC 15
+
+// Defines for task manager and such
+#define CALTDEL LCTL(LALT(KC_DEL))
+#define TSKMGR LCTL(LSFT(KC_ESC))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /* Qwerty
+     * ,-------------.
+     * |   1  |  2   |
+     * |------+------|
+     * |   3  |  4   |
+     * |------+------|
+     * |   5  |  6   |
+     * |------+------|
+     * | FUNC | RGB  |
+     * `-------------'
+     */
+    [_QWERTY] = LAYOUT(KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, MO(_FUNC), TG(_RGB)),
```

Comment:
The layers *could* be formatted to actually look like the layout (four rows of two keycodes), but I consider that optional.

Reply:
updated, must have missed it getting reformatted